### PR TITLE
JBPM-7850 - KIE Server form rendering - support for multi sub forms

### PR DESCRIPTION
**Pull request by mswiderski**
@tsurdilo @MarianMacik  please have a look, [here](https://www.youtube.com/watch?v=7pQV63LaQn4) is a short screen cast showing it in action

### CODE REVIEW

**Comment by tsurdilo** [comment]
@mswiderski watched video and tried in business app - works great and nice video too!

**Comment by mswiderski** [comment]
Jenkins retest this

File: kie-server-parent/kie-server-tests/kie-server-integ-tests-all/src/test/java/org/kie/server/integrationtests/jbpm/RenderFormServiceIntegrationTest.java [hunk 1]
```diff
@@ -430,12 +447,311 @@ public void testRenderCaseFormViaUIClientTestBootstrap() throws Exception {
...
+        String result = uiServicesClient.renderProcessForm(CONTAINER_ID, PLACE_ORDER_PROCESS_ID);
+        logger.debug("Form content is '{}'", result);
+        assertThat(result).isNotNull().isNotEmpty();
+        
+        // it has the patternfly (default renderer) css
+        assertThat(result).contains("/files/patternfly/css/patternfly.min.css\" rel=\"stylesheet\">");
+        assertThat(result).contains("/files/patternfly/css/patternfly-additions.min.css\" rel=\"stylesheet\">");
+        
+        // it has required js files
+        assertThat(result).contains("/files/patternfly/js/jquery.min.js\"></script>");
```
Comment (rsynek) [inline kie-server-parent/kie-server-tests/kie-server-integ-tests-all/src/test/java/org/kie/server/integrationtests/jbpm/RenderFormServiceIntegrationTest.java]: A suggestion - with AssertJ you can chain the assertions over a single object (in this case `result`) in a builder-like style without repeating `assertThat` on every line.

**Comment by mswiderski** [comment]
Jenkins retest this

**Comment by mswiderski** [comment]
Jenkins retest this

**Comment by mswiderski** [comment]
Jenkins retest this

**Comment by mswiderski** [comment]
Jenkins retest this

**Comment by mswiderski** [comment]
Jenkins retest this

**Pull request merged**